api/catalog/management/commands/generatewaveforms.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import subprocess
+
+from catalog.api.models.audio import Audio
+from django_tqdm import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Generates waveforms for all audio records to populate the cache"
+
+    def handle(self, *args, **options):
+        # These logs really muck up the tqdm output and don't give us much helpful
+        # information, so they get silenced
+        logging.getLogger("catalog.api.utils.waveform").setLevel(logging.WARNING)
+
+        audios = Audio.objects.all().order_by("id")
+        count = audios.count()
+        self.stdout.write(
+            self.style.NOTICE(f"Generating waveforms for {count:,} records")
+        )
+        with self.tqdm(total=count) as progress:
+            for audio in audios:
+                try:
+                    audio.get_or_create_waveform()
```

Comment:
I would be okay with adding a delay at this step :slightly_smiling_face: 0.25 seconds is a small fraction of the time it takes to actually retrieve + compute the waveform, so that seems fine.

Reply:
I was noticing that as I ran it locally. I _almost_ suggested doing something like a speed _limit_ (1 row per second but include the time spent processing the previous row in the "wait" time between each row... or something?) but decided that was probably too complicated/overkill (though if there's some kind of utility for already doing that easily it would be sweet to use it).